src/main/resources/config/application.yml

```diff
@@ -20,6 +19,8 @@ server:
     mime-types: application/json,application/xml,text/html,text/xml,text/plain
 
 revocation:
+  authorization:
+    reload: 600000
```

Comment:
If a new revocation rule is going to be inserted, it means a new application will manage revocations/a new type of revocation which wasn't allowed before is going to be allowed. And adding the new rule is a manual process anyway, that's why we thought 10 minutes was reasonable.